dvc/remote/ssh/__init__.py

```diff
@@ -47,9 +50,21 @@ class RemoteSSH(RemoteBASE):
     # We use conservative setting of 4 instead to not exhaust max sessions.
     CHECKSUM_JOBS = 4
 
+    # XXX: Friendly reminder to add "reflink" to the default types
```

Comment:
let's not use XXX and TODO in code. Let's create a ticket if something needs to be updated at some point in the future.

Reply:
this is a reminder for myself, @shcheklein , I'll make sure to remove them when implemented :slightly_smiling_face: